config/interpolate_funcs.go

```diff
@@ -640,6 +641,44 @@ func appendIfMissing(slice []string, element string) []string {
 	return append(slice, element)
 }
 
+// interpolationFuncFilter implements the "filter" function that
+// removes elements from a list
+func interpolationFuncFilter() ast.Function {
+	return ast.Function{
+		ArgTypes:   []ast.Type{ast.TypeList, ast.TypeList, ast.TypeString},
+		ReturnType: ast.TypeList,
+		Callback: func(args []interface{}) (interface{}, error) {
+			var list []string
+
+			argument, _ := args[0].([]ast.Variable)
+			keys, _ := args[1].([]ast.Variable)
+			searchkey, _ := args[2].(string)
```

Comment:
Is there any specific reason you decided to silence the errors here? What if the user misreads or just doesn't read the docs, or just simply makes a mistake in providing the right argument types?